ortac-core.opam

```diff
@@ -1,16 +1,35 @@
 opam-version: "2.0"
-synopsis: "OCaml Runtime Assertion Checking based on Gospel"
-description: "OCaml Runtime Assertion Checking based on Gospel"
-maintainer: ["Clément Pascutto <clement@pascutto.fr>"]
+synopsis:
+  "Ortac (OCaml Runtime Assertion Checking based on Gospel) core tool and library"
```

Comment:
That's the only place where "based on Gospel" is present inside the parenthesis.
I would remove them as this is not part of the acronym.